src/index.html

```diff
@@ -2,15 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchForm"
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <label>
+        <input type="text"
+        name="searchInputBig"
+        class="searchInput"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress" required
+      >
+      </label>
+      <button type="submit"
+       class="searchIcon">
+      </button>
+    </form>
+
+    <form
+      class="searchForm searchFormSmall"
+      action="/"
+      method="get"
+      data-qa="small"
+    >
+      <label>
+        <input type="text"
+        name="searchInputSmall"
```

Comment:
Please don't forget check your formatting(should be 2-space indentation)
![image](https://user-images.githubusercontent.com/54936044/228686799-c19751cb-97d9-4ca1-8980-6663ec123e40.png)